packages/teleport/src/Teleport.tsx

```diff
@@ -109,4 +120,6 @@ export default hot(Teleport);
 export type Props = {
   ctx: TeleportContext;
   history: History;
+  renderPublicRoutes?(children?: any): any;
+  renderPrivateRoutes?(CustomMain?: any): any;
```

Comment:
I think these type should be more specific, ie `renderPublicRoutes` probably can't accept like say an array of strings